docs/pages/eas-update/code-signing.mdx

```diff
@@ -4,52 +4,64 @@ description: Learn how code signing and key rotation work in EAS Update.
 ---
 
 import Head from '~/components/Head';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
 
 <Head title="EAS Update Code Signing" />
 
-> EAS Update Code Signing is only available to accounts subscribed to the EAS Enterprise plan. [Sign up](https://expo.dev/pricing).
+> **info** EAS Update Code Signing is only available to accounts subscribed to the EAS Enterprise plan. [Sign up](https://expo.dev/pricing).
```

Comment:
```suggestion
> **info** EAS Update Code Signing is only available to accounts subscribed to the [EAS Enterprise plan](https://expo.dev/pricing).
```

Since, we are not linking to the Sign up page, above suggestion makes more sense to me for linking.

Reply:
Like that idea! 👌